mmv1/products/discoveryengine/DataStore.yaml

```diff
@@ -24,7 +24,7 @@ references: !ruby/object:Api::Resource::ReferenceLinks
 
 base_url: 'projects/{{project}}/locations/{{location}}/collections/default_collection/dataStores'
 self_link: 'projects/{{project}}/locations/{{location}}/collections/default_collection/dataStores/{{data_store_id}}'
-create_url: 'projects/{{project}}/locations/{{location}}/collections/default_collection/dataStores?dataStoreId={{data_store_id}}&createAdvancedSiteSearch={{create_advanced_site_search}}'
+create_url: 'projects/{{project}}/locations/{{location}}/collections/default_collection/dataStores?dataStoreId={{data_store_id}}&createAdvancedSiteSearch={{create_advanced_site_search}}&skipDefaultSchemaCreation={{skip_default_schema_creation}}'
```

Comment:
could you provide more info as to where this parameter is coming from? I'm wanting to get more info to understand why the update_test is failing but I can't seem to find it in the docs: https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1/projects.locations.collections.dataStores

Reply:
Looks like the update test is new and is actively being worked on: https://github.com/GoogleCloudPlatform/magic-modules/pull/11363

Let's wait for this to be fixed, once it's merged you can rebase this PR and this once more before approving.

Reply:
@jialei-chen can you rebase now? The PR that was previously mentioned has been merged.